config/routes.rb

```diff
@@ -372,6 +372,7 @@ def scoped_navigation_routes(context, navigation)
 
     mount ActionCable.server => '/cable'
     get '/.well-known/pki-validation/:id', to: 'public_pages#pki_validation'
+    get '/.well-known/acme-challenge/:id', to: 'public_pages#acme_challenge'
```

Comment:
i think if you want this to be served on the FYST domain you would need it to be in the third and final constraints block in the routes file instead

Reply:
Good catch! Done